### DISCUSSION TITLE
Riru flavor meaning?

### DISCUSSION THREAD

**Discussion by pritster5** [LSPosed/LSPosed#2611] [Q&A]
Does the default version of this framework use Zygisk? 

If so, we should skip step 2 in the instructions right? Since Riru is outdated and Zygisk is the way to go now?

**Comment by yujincheng08** [comment]
Define default version.

**Reply by pritster5** [reply]
Since the guide specifically calls out the riru version as "(for riru flavor)", it's implying that Riru is optional/or not the norm. So the default is probably something that doesn't use Riru i.e. Zygisk.

**Reply by yujincheng08** [reply]
good imagination.

**Reply by pritster5** [reply]
This is how English works lol. 

E.g. 

1. Make food
2.  (For pepper flavor) add pepper
3. Eat food
If I don't add pepper, there's still food there i.e. the default.  In this case, I was asking if that was the Zygisk version of the framework.

 To be clearer, which version of this is the recommended version? Zygisk or Riru?
